pandas/core/internals/blocks.py

```diff
@@ -1972,6 +1972,14 @@ class ObjectValuesExtensionBlock(ExtensionBlock):
     Series[T].values is an ndarray of objects.
     """
 
+    def diff(self, n: int, axis: int = 1) -> List["Block"]:
+        # Block.shape vs. Block.values.shape mismatch
```

Comment:
you are supposed to use _block_shape here, that's the point of it.

Reply:
is this possilbe?

Reply:
are you asking if it is possible to have `block.shape != block.values.shape`?  Yes, and it is the cause of kludges all over the code, which is why i want 2D EAs.